session_analyzer.py

```diff
@@ -13,6 +14,7 @@ def __init__(self):
     @asyncio.coroutine
     def analyze(self):
         while True:
+            time.sleep(1)
```

Comment:
Have a look at this: https://docs.python.org/3/library/asyncio-task.html#example-coroutine-displaying-the-current-date Which version of Python are you using?